geode-lucene/src/upgradeTest/java/org/apache/geode/cache/lucene/RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated.java

```diff
@@ -31,6 +32,7 @@
 public class RollingUpgradeQueryReturnsCorrectResultsAfterClientAndServersAreRolledOverAllBucketsCreated
     extends LuceneSearchWithRollingUpgradeDUnit {
 
+  @Ignore("Not supported")
```

Comment:
I don't think we should add an ignore to this test... the test comments explicitly state it's to catch a problem from going from 6 -> 7 (which this PR does)

Reply:
Hi @jhuynh1 ,
As we agreed on dev list we need to **do query after all servers are upgraded**, but this is the case where one server is upgraded, lucene index created on it and then shutdown. Server with older version will get _IndexFormatTooNewException_ as lucene 7.x has different index format and it cannot read it. 
In that case this test is not supported.

Reply:
Was there a modified version of this test to show the end result works?  Just to prove that rolling still allows us to query older indexed data at the end of the rolling upgrade?  I thought there was some work/test added to that feature branch?

Reply:
Should there also be some doc change and a test added so that makes sure queries are "stuck" if versions differ?  My fear/concern is that a change then opens up this "problem" in the future.

Reply:
I think that we can add to _Apache Lucene Integration_ in _Requirements and Caveats_ that executing queries is possible only when rolling upgrade is finished.
Also we can try to write a test to upgrade one of two server, execute query and except that query will failed as you cannot execute it before all members are rolled.